pages/learn/deploy/deploy-with-balena-button.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Deploy with {{ $names.company.lower }} button
+excerpt: The Deploy with {{ $names.company.lower }} button allows users to do a single-click deployment and configuration of an application to {{ $names.cloud.lower }} 
+---
+
+# Deploy with {{ $names.company.lower }} button
+
+The **Deploy with {{ $names.company.lower }}** button allows users to perform a single-click deployment and configuration of an application to {{ $names.cloud.lower }}.
+
+![Deploy with {{ $names.company.lower }}](/img/configuration/deploy-to-balena-button.png)
```

Comment:
Let's use `https://balena.io/deploy.png` instead, so we have a single centralized place where the image exists.